src/index.html

```diff
@@ -5,8 +5,372 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="" class="header__link">
+        <img
+          src="images/logo.png"
+          alt="logo Moyo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li"><a href="" class="nav__link nav__link--is-active is-active">APPLE</a></li>
+          <li class="nav__li"><a href="" class="nav__link">SAMSUNG</a></li>
+          <li class="nav__li"><a href="" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+          <li class="nav__li">
+            <a
+              href=""
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__li"><a href="" class="nav__link">ГАДЖЕТЫ</a></li>
+          <li class="nav__li"><a href="" class="nav__link">ПЛАНШЕТЫ</a></li>
+          <li class="nav__li"><a href="" class="nav__link">ФОТО</a></li>
+          <li class="nav__li"><a href="" class="nav__link">ВИДЕО</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__first">
```

Comment:
```suggestion
      <div class="main__container container">
```

Reply:
no need for 2 separate divs for cards, there should be ONE container for all cards
read about flex wrap property